app/controllers/automated_workflow_controller.rb

```diff
@@ -0,0 +1,38 @@
+class AutomatedWorkflowController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericListMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
+  include Mixins::BreadcrumbsMixin
+
+  def self.model
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook
```

Comment:
@Fryguy , I don't know which model has to be used here. For now I had used Playbook just so that to see that the page works till here..

Reply:
`ManageIQ::Providers::Workflows::AutomationManager::Workflow`

Reply:
Every list view screen requires a report file which defines the table structure - the name of the file matches the model.